worker/buildbot_worker/test/unit/test_bot.py

```diff
@@ -68,11 +68,7 @@ def setUp(self) -> None:
 PRETTY_NAME="Test 1.0 Generic"
 VERSION_ID="1"
 """)
-        self.real_bot = pb.BotPbLike(
```

Comment:
In commit message of `worker: fix BotPbLike instantiate in TestBot` better refer to the commit (ed677db33fb54354c75674f4bbdac200cd0e40fc), not link on github.